lib/internal/crypto/random.js

```diff
@@ -314,81 +340,27 @@ function getBufferedUUID() {
 
   if (uuidBatch === 0) randomFillSync(uuidData);
   uuidBatch = (uuidBatch + 1) % kBatchSize;
-  return uuidData.slice(uuidBatch * 16, (uuidBatch * 16) + 16);
+  return serializeUUID(uuidData, uuidBatch * 16);
+}
+
+function getUnbufferedUUID() {
+  let uuidBuf = uuidNotBuffered;
+  if (uuidBuf === undefined)
+    uuidBuf = uuidNotBuffered = secureBuffer(16);
+  if (uuidBuf === undefined)
+    throw new ERR_OPERATION_FAILED('Out of memory');
+  randomFillSync(uuidBuf);
+  return serializeUUID(uuidBuf);
 }
 
 function randomUUID(options) {
   if (options !== undefined)
     validateObject(options, 'options');
-  const {
-    disableEntropyCache = false,
-  } = { ...options };
+  const { disableEntropyCache = false } = options || {};
```

Comment:
Style nit (makes it easier to add additional options later if necessary)

```suggestion
  const {
    disableEntropyCache = false,
  } = options || {};
```